src/content/docs/kubernetes-pixie/kubernetes-integration/installation/upgrading-kubernetes.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Upgrading Kubernetes integration'
```

Comment:
@nbaenam I think "the" would be good in the title: "Upgrading the Kubernetes integration."